http/services/todoService.js

```diff
@@ -1,7 +1,21 @@
-import HttpService from "./httpService";
+import BaseService from "./baseService";
+import Cookie from "js-cookie";
 
-class TodoService extends HttpService {
+const ENDPOINTS = {
+    GET_TODOS: (id) => `/users/${id}/todos`,
+};
+
+class TodoService extends BaseService {
     constructor() {
         super();
+
+        this.getTodos = this.getTodos.bind(this);
+    }
+
+    async getTodos(payload) {
+        const todos = await this.client.get(ENDPOINTS.GET_TODOS(payload));
+        return todos;
```

Comment:
mozes samo getTodos = payload => this.client.get(ENDPOINTS.GET_TODOS(payload)); 
ovo se poziva iz sage koje su generatorske fje i one ce cekati da se ovaj promise resolvuje